Documentation/api.md

```diff
@@ -146,7 +146,7 @@ The `ClusterMonitoringConfiguration` resource defines settings that customize th
 
 #### Description
 
-The `K8sPrometheusAdapter` resource defines settings for the Prometheus Adapter component.
+The `K8sPrometheusAdapter` resource defines settings for the Prometheus Adapter component. This is deprecated and will be removed in a future version, setting it has no effect.
```

Comment:
It depends on #2386, if merged the config might still have an effect.

Reply:
Shouldn't we say it as deprecated so that its not used in future. We want to use the config value only if it exists right

Reply:
I can re-word this